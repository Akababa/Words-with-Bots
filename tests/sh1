pm damned 5 6 d 4
pm bout 6 7 d
pm letters 9 5
pm noirs 6 10 d
pm rale 3 11 d
pm skiver 3 6
pm qis 1 6 d
pm joe 7 9 d
pm daze 5 6
pm hep 2 10 d
pm qi 1 6

ra +fy[ewsa
lm
pr b