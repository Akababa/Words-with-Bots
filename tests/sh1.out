rack: FY[EWSA
NEWSLETTERS @ 9, 1 acr for 63 (hscore:63) (blank:1)
     FAQIRS @ 1, 4 acr for 51 (hscore:51) (blank:5)
      FIZZY @ 2, 8 dwn for 26 (hscore:26) (blank:3)
        VAV @ 1, 9 dwn for 22 (hscore:22) (blank:1)
        TAV @ 1, 9 dwn for 22 (hscore:22) (blank:1)
        REV @ 1, 9 dwn for 22 (hscore:22) (blank:1)
        LEV @ 1, 9 dwn for 22 (hscore:22) (blank:1)
        LAV @ 1, 9 dwn for 22 (hscore:22) (blank:1)
        DEV @ 1, 9 dwn for 22 (hscore:22) (blank:1)
        AYS @10, 8 acr for 22 (hscore:22)
        AYS @10, 8 acr for 20 (hscore:20) (blank:1)
        ASS @10, 8 acr for 18 (hscore:18)
         ZA @ 5, 8 dwn for 17 (hscore:17)
       BABA @ 6, 5 acr for 17 (hscore:17) (blank:1)
        ABA @ 6, 6 acr for 17 (hscore:17)
      SAYID @10, 2 acr for 16 (hscore:16) (blank:4)
      FAQIR @ 1, 4 acr for 16 (hscore:16) (blank:5)
        AYS @10, 8 acr for 16 (hscore:16) (blank:2)
        ASS @10, 8 acr for 16 (hscore:16) (blank:2)
        ASS @10, 8 acr for 16 (hscore:16) (blank:1)
         ZA @ 5, 8 dwn for 15 (hscore:15) (blank:2)
      WEIRS @ 2, 4 acr for 15 (hscore:15) (blank:4)
      WAIRS @ 2, 4 acr for 15 (hscore:15) (blank:4)
      WAINS @ 2, 4 acr for 15 (hscore:15) (blank:4)
      WAILS @ 2, 4 acr for 15 (hscore:15) (blank:4)
        SEI @ 1, 8 dwn for 15 (hscore:15)
       FIZZ @ 2, 8 dwn for 15 (hscore:15) (blank:3)
      FAIRS @ 2, 4 acr for 15 (hscore:15) (blank:4)
      FAILS @ 2, 4 acr for 15 (hscore:15) (blank:4)
        ABA @ 6, 6 acr for 15 (hscore:15) (blank:3)
       WINS @ 2, 5 acr for 14 (hscore:14) (blank:3)
      WHAMO @ 7, 3 acr for 14 (hscore:14) (blank:2)
      SWAMI @ 2, 2 acr for 14 (hscore:14) (blank:4)
        SRI @ 1, 8 dwn for 14 (hscore:14) (blank:2)
        SOW @ 9,11 dwn for 14 (hscore:14) (blank:2)
        SKI @ 1, 8 dwn for 14 (hscore:14) (blank:2)
        SEI @ 1, 8 dwn for 14 (hscore:14) (blank:2)
       FIRS @ 2, 5 acr for 14 (hscore:14) (blank:3)
       FINS @ 2, 5 acr for 14 (hscore:14) (blank:3)
       FILS @ 2, 5 acr for 14 (hscore:14) (blank:3)
       FILA @ 2, 5 acr for 14 (hscore:14) (blank:3)
      AYINS @ 2, 4 acr for 14 (hscore:14) (blank:4)
       YINS @ 2, 5 acr for 13 (hscore:13) (blank:3)
      SWAIN @ 2, 3 acr for 13 (hscore:13) (blank:5)
      SWAIL @ 2, 3 acr for 13 (hscore:13) (blank:5)
      STEAD @10, 2 acr for 13 (hscore:13) (blank:2)
        SEI @ 1, 8 dwn for 13 (hscore:13) (blank:1)
       GLEY @ 8, 5 dwn for 13 (hscore:13) (blank:1)
       WEIR @ 2, 4 acr for 12 (hscore:12) (blank:4)
       WAIR @ 2, 4 acr for 12 (hscore:12) (blank:4)
       WAIN @ 2, 4 acr for 12 (hscore:12) (blank:4)
       WAIL @ 2, 4 acr for 12 (hscore:12) (blank:4)
        QIS @ 1, 6 acr for 12 (hscore:12)
       FAIR @ 2, 4 acr for 12 (hscore:12) (blank:4)
       FAIN @ 2, 4 acr for 12 (hscore:12) (blank:4)
       FAIL @ 2, 4 acr for 12 (hscore:12) (blank:4)
        WIN @ 2, 5 acr for 11 (hscore:11) (blank:3)
        SOY @ 9,11 dwn for 11 (hscore:11) (blank:2)
        SHY @ 9,11 dwn for 11 (hscore:11) (blank:2)
        QIS @ 1, 6 acr for 11 (hscore:11) (blank:3)
        FIR @ 2, 5 acr for 11 (hscore:11) (blank:3)
        FIN @ 2, 5 acr for 11 (hscore:11) (blank:3)
        FIL @ 2, 5 acr for 11 (hscore:11) (blank:3)
       AYIN @ 2, 4 acr for 11 (hscore:11) (blank:4)
       AIRS @ 2, 5 acr for 11 (hscore:11) (blank:3)
       AINS @ 2, 5 acr for 11 (hscore:11) (blank:3)
       AILS @ 2, 5 acr for 11 (hscore:11) (blank:3)
        YIN @ 2, 5 acr for 10 (hscore:10) (blank:3)
       WOAD @10, 3 acr for 10 (hscore:10) (blank:2)
        WAD @10, 4 acr for 10 (hscore:10)
       SWAB @ 6, 4 acr for 10 (hscore:10)
      NAWAB @ 6, 3 acr for 10 (hscore:10) (blank:1)
        LAW @ 9, 5 dwn for 10 (hscore:10)
        INS @ 2, 6 acr for 10 (hscore:10) (blank:2)
        FAD @10, 4 acr for 10 (hscore:10)
       WIFI @ 2, 3 acr for 9 (hscore:9) (blank:2)
        WAB @ 6, 5 acr for 9 (hscore:9)
       SWAB @ 6, 4 acr for 9 (hscore:9) (blank:1)
       SAIN @ 2, 4 acr for 9 (hscore:9) (blank:4)
       SAIL @ 2, 4 acr for 9 (hscore:9) (blank:4)
        LEY @ 9, 5 dwn for 9 (hscore:9)
        LAY @ 9, 5 dwn for 9 (hscore:9)
       FLAB @ 6, 4 acr for 9 (hscore:9) (blank:2)
       FEOD @10, 3 acr for 9 (hscore:9) (blank:3)
        FAB @ 6, 5 acr for 9 (hscore:9)
       DYAD @10, 3 acr for 9 (hscore:9) (blank:1)
        WAD @10, 4 acr for 8 (hscore:8) (blank:2)
        SIR @ 2, 5 acr for 8 (hscore:8) (blank:3)
        SIN @ 2, 5 acr for 8 (hscore:8) (blank:3)
        SIC @ 2, 5 acr for 8 (hscore:8) (blank:3)
      SERAI @ 2, 2 acr for 8 (hscore:8) (blank:3)
  RELETTERS @ 9, 3 acr for 8 (hscore:8) (blank:1)
       MENU @ 8, 4 acr for 8 (hscore:8) (blank:1)
        LOW @ 9, 5 dwn for 8 (hscore:8) (blank:2)
        LAW @ 9, 5 dwn for 8 (hscore:8) (blank:2)
       GENU @ 8, 4 acr for 8 (hscore:8) (blank:1)
        FID @10, 4 acr for 8 (hscore:8) (blank:2)
        FAD @10, 4 acr for 8 (hscore:8) (blank:2)
        AIR @ 2, 5 acr for 8 (hscore:8) (blank:3)
        AIN @ 2, 5 acr for 8 (hscore:8) (blank:3)
        AIL @ 2, 5 acr for 8 (hscore:8) (blank:3)
        YOD @10, 4 acr for 7 (hscore:7) (blank:2)
        YID @10, 4 acr for 7 (hscore:7) (blank:2)
       SHAD @10, 3 acr for 7 (hscore:7) (blank:2)
       SCAD @10, 3 acr for 7 (hscore:7) (blank:2)
        SAD @10, 4 acr for 7 (hscore:7)
       READ @10, 3 acr for 7 (hscore:7) (blank:1)
       MEAD @10, 3 acr for 7 (hscore:7) (blank:1)
        LEY @ 9, 5 dwn for 7 (hscore:7) (blank:2)
       LEAD @10, 3 acr for 7 (hscore:7) (blank:1)
        LEA @ 9, 5 dwn for 7 (hscore:7)
        LAY @ 9, 5 dwn for 7 (hscore:7) (blank:2)
        LAS @ 9, 5 dwn for 7 (hscore:7)
        INK @ 1, 7 dwn for 7 (hscore:7) (blank:2)
         IN @ 2, 6 acr for 7 (hscore:7) (blank:2)
       HEAD @10, 3 acr for 7 (hscore:7) (blank:1)
       EGAD @10, 3 acr for 7 (hscore:7) (blank:2)
       DEAD @10, 3 acr for 7 (hscore:7) (blank:1)
       BEAD @10, 3 acr for 7 (hscore:7) (blank:1)
       WADI @ 2, 3 acr for 6 (hscore:6) (blank:3)
        WAD @10, 4 acr for 6 (hscore:6) (blank:1)
        TWA @ 9, 8 dwn for 6 (hscore:6)
        TEW @ 9, 8 dwn for 6 (hscore:6)
        TAW @ 9, 8 dwn for 6 (hscore:6)
        TAD @10, 4 acr for 6 (hscore:6) (blank:1)
       SWAB @ 6, 4 acr for 6 (hscore:6) (blank:2)
       SUMO @ 7, 4 acr for 6 (hscore:6) (blank:2)
       STAB @ 6, 4 acr for 6 (hscore:6) (blank:2)
       SLAB @ 6, 4 acr for 6 (hscore:6) (blank:2)
       SHMO @ 7, 4 acr for 6 (hscore:6) (blank:2)
       SCAB @ 6, 4 acr for 6 (hscore:6) (blank:2)
       SAID @10, 3 acr for 6 (hscore:6) (blank:3)
        SAD @10, 4 acr for 6 (hscore:6) (blank:1)
        SAB @ 6, 5 acr for 6 (hscore:6)
        RAD @10, 4 acr for 6 (hscore:6) (blank:1)
        PAD @10, 4 acr for 6 (hscore:6) (blank:1)
       MEMO @ 7, 4 acr for 6 (hscore:6) (blank:1)
        MAD @10, 4 acr for 6 (hscore:6) (blank:1)
       LWEI @ 2, 3 acr for 6 (hscore:6) (blank:1)
        LEZ @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LEY @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LEX @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LEV @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LEU @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LET @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LEK @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LEI @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LEG @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LEE @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LED @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LEA @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LAY @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LAX @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LAW @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LAV @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LAT @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LAS @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LAR @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LAP @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LAM @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LAG @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LAD @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LAD @10, 4 acr for 6 (hscore:6) (blank:1)
        LAC @ 9, 5 dwn for 6 (hscore:6) (blank:3)
        LAB @ 9, 5 dwn for 6 (hscore:6) (blank:3)
         LA @ 9, 5 dwn for 6 (hscore:6)
        HAD @10, 4 acr for 6 (hscore:6) (blank:1)
        GAD @10, 4 acr for 6 (hscore:6) (blank:1)
        FAD @10, 4 acr for 6 (hscore:6) (blank:1)
       DEMO @ 7, 4 acr for 6 (hscore:6) (blank:1)
       DEFI @ 2, 3 acr for 6 (hscore:6) (blank:1)
        DAD @10, 4 acr for 6 (hscore:6) (blank:1)
       CAMO @ 7, 4 acr for 6 (hscore:6) (blank:1)
        CAD @10, 4 acr for 6 (hscore:6) (blank:1)
        BAD @10, 4 acr for 6 (hscore:6) (blank:1)
       AMMO @ 7, 4 acr for 6 (hscore:6) (blank:2)
         AD @10, 5 acr for 6 (hscore:6)
       YETI @ 2, 3 acr for 5 (hscore:5) (blank:3)
       YAGI @ 2, 3 acr for 5 (hscore:5) (blank:3)
        WAB @ 6, 5 acr for 5 (hscore:5) (blank:1)
        TYE @ 9, 8 dwn for 5 (hscore:5)
        TWO @ 9, 8 dwn for 5 (hscore:5) (blank:3)
        TWA @ 9, 8 dwn for 5 (hscore:5) (blank:3)
        TOW @ 9, 8 dwn for 5 (hscore:5) (blank:2)
        TEW @ 9, 8 dwn for 5 (hscore:5) (blank:2)
        TAW @ 9, 8 dwn for 5 (hscore:5) (blank:2)
        TAB @ 6, 5 acr for 5 (hscore:5) (blank:1)
       SYLI @ 2, 3 acr for 5 (hscore:5) (blank:3)
        SOS @ 9,11 dwn for 5 (hscore:5) (blank:2)
        SOD @10, 4 acr for 5 (hscore:5) (blank:2)
        SIS @ 9,11 dwn for 5 (hscore:5) (blank:2)
        SHE @ 9,11 dwn for 5 (hscore:5) (blank:2)
        SHA @ 9,11 dwn for 5 (hscore:5) (blank:2)
        SAD @10, 4 acr for 5 (hscore:5) (blank:2)
        SAB @ 6, 5 acr for 5 (hscore:5) (blank:1)
        NAB @ 6, 5 acr for 5 (hscore:5) (blank:1)
        LIS @ 9, 5 dwn for 5 (hscore:5) (blank:2)
        LIE @ 9, 5 dwn for 5 (hscore:5) (blank:2)
        LEE @ 9, 5 dwn for 5 (hscore:5) (blank:2)
        LEA @ 9, 5 dwn for 5 (hscore:5) (blank:2)
        LAS @ 9, 5 dwn for 5 (hscore:5) (blank:2)
        LAB @ 6, 5 acr for 5 (hscore:5) (blank:1)
        KAB @ 6, 5 acr for 5 (hscore:5) (blank:1)
        JAB @ 6, 5 acr for 5 (hscore:5) (blank:1)
        GAB @ 6, 5 acr for 5 (hscore:5) (blank:1)
         FI @ 2, 8 dwn for 5 (hscore:5)
         FI @ 2, 5 acr for 5 (hscore:5)
        FAB @ 6, 5 acr for 5 (hscore:5) (blank:1)
        DAB @ 6, 5 acr for 5 (hscore:5) (blank:1)
        CAB @ 6, 5 acr for 5 (hscore:5) (blank:1)
        AID @10, 4 acr for 5 (hscore:5) (blank:2)
        TYE @ 9, 8 dwn for 4 (hscore:4) (blank:3)
        TRY @ 9, 8 dwn for 4 (hscore:4) (blank:2)
        TOY @ 9, 8 dwn for 4 (hscore:4) (blank:2)
        THY @ 9, 8 dwn for 4 (hscore:4) (blank:2)
         OD @10, 5 acr for 4 (hscore:4) (blank:1)
         LO @ 9, 5 dwn for 4 (hscore:4) (blank:2)
         LI @ 9, 5 dwn for 4 (hscore:4) (blank:2)
         LA @ 9, 5 dwn for 4 (hscore:4) (blank:2)
         ID @10, 5 acr for 4 (hscore:4) (blank:1)
         AD @10, 5 acr for 4 (hscore:4) (blank:1)
        TEA @ 9, 8 dwn for 3 (hscore:3)
        TAS @ 9, 8 dwn for 3 (hscore:3)
        TAE @ 9, 8 dwn for 3 (hscore:3)
       SEMI @ 2, 3 acr for 3 (hscore:3) (blank:3)
        SEI @ 2, 4 acr for 3 (hscore:3)
       SATI @ 2, 3 acr for 3 (hscore:3) (blank:3)
       SARI @ 2, 3 acr for 3 (hscore:3) (blank:3)
       SAKI @ 2, 3 acr for 3 (hscore:3) (blank:3)
       SADI @ 2, 3 acr for 3 (hscore:3) (blank:3)
       ASCI @ 2, 3 acr for 3 (hscore:3) (blank:3)
        TYE @ 9, 8 dwn for 2 (hscore:2) (blank:2)
        TWA @ 9, 8 dwn for 2 (hscore:2) (blank:2)
        TSK @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TOE @ 9, 8 dwn for 2 (hscore:2) (blank:2)
        TIS @ 9, 8 dwn for 2 (hscore:2) (blank:2)
        TIE @ 9, 8 dwn for 2 (hscore:2) (blank:2)
        THE @ 9, 8 dwn for 2 (hscore:2) (blank:2)
        TEW @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TET @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TEN @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TEL @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TEG @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TEE @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TEE @ 9, 8 dwn for 2 (hscore:2) (blank:2)
        TED @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TEA @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TEA @ 9, 8 dwn for 2 (hscore:2) (blank:2)
        TAX @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAW @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAV @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAU @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAT @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAS @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAS @ 9, 8 dwn for 2 (hscore:2) (blank:2)
        TAR @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAP @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAO @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAN @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAM @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAJ @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAG @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAE @ 9, 8 dwn for 2 (hscore:2) (blank:2)
        TAE @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAD @ 9, 8 dwn for 2 (hscore:2) (blank:3)
        TAB @ 9, 8 dwn for 2 (hscore:2) (blank:3)
         TA @ 9, 8 dwn for 2 (hscore:2)
        SRI @ 2, 4 acr for 2 (hscore:2) (blank:2)
         SO @10,10 acr for 2 (hscore:2) (blank:2)
         SO @ 9,11 dwn for 2 (hscore:2) (blank:2)
        SKI @ 2, 4 acr for 2 (hscore:2) (blank:2)
         SI @ 2, 5 acr for 2 (hscore:2)
         SI @ 2, 8 dwn for 2 (hscore:2)
         SI @10,10 acr for 2 (hscore:2) (blank:2)
         SI @ 9,11 dwn for 2 (hscore:2) (blank:2)
         SH @ 9,11 dwn for 2 (hscore:2) (blank:2)
         SH @10,10 acr for 2 (hscore:2) (blank:2)
        SEI @ 2, 4 acr for 2 (hscore:2) (blank:2)
        SEI @ 2, 4 acr for 2 (hscore:2) (blank:1)
        REI @ 2, 4 acr for 2 (hscore:2) (blank:1)
        RAI @ 2, 4 acr for 2 (hscore:2) (blank:1)
        PSI @ 2, 4 acr for 2 (hscore:2) (blank:1)
        LEI @ 2, 4 acr for 2 (hscore:2) (blank:1)
        ANI @ 2, 4 acr for 2 (hscore:2) (blank:2)
        AMI @ 2, 4 acr for 2 (hscore:2) (blank:2)
         AI @ 2, 5 acr for 2 (hscore:2)
         AI @ 2, 8 dwn for 2 (hscore:2)
        AHI @ 2, 4 acr for 2 (hscore:2) (blank:2)
         XI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         XI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         TO @ 9, 8 dwn for 1 (hscore:1) (blank:2)
         TI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         TI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         TI @ 9, 8 dwn for 1 (hscore:1) (blank:2)
         TA @ 9, 8 dwn for 1 (hscore:1) (blank:2)
         SI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         SI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         QI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         QI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         PI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         PI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         OI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         OI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         MI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         MI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         LI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         LI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         KI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         KI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         HI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         HI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         GI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         GI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         FI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         FI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         DI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         DI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         BI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         BI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
         AI @ 2, 5 acr for 1 (hscore:1) (blank:1)
         AI @ 2, 8 dwn for 1 (hscore:1) (blank:1)
board:
012345678901
1*.!..QI.!.*
2.?...I...H.
3!.*.'SKIVER
4...*...*.PA
5..'..DAZE.L
6.?...AB..NE
7..'..MO.JO.
8...*.NU*OI.
9!.*.LETTERS
0.?...D...S.
1*.!.....!.*
